pages/ugm2023.md

```diff
@@ -70,6 +70,25 @@ There are many other hotel and guest house options accesible to and from the ven
 
 Citizens of several countries may travel to Portugal to attend a meeting without obtaining a visa. Please visit the [Portugal Ministry of Foreign Affairs website][mne.gov.pt] for more information about travel visas. Confirmed attendees requiring a letter clarifying the purpose of travel should [contact the MDAnalysis project manager][email]. 
 
+## Health and Safety
+We are committed to providing a safe, comfortable, and healthy environment for the MDAnalysis UGM and have looked to local and international entities and other organziations in the open source community to develop onsite safety measures for UGM attendees.
+
+### Vaccine Policy
+MDAnalysis strongly recommends that UGM attendees remain current (i.e., receiving all recommended primary and booster doses recommended by your local government) with their COVID-19 vaccinations as a personal protective measure. 
+
+### Testing
+A COVID-19 test result is not required to determine whether you can attend the UGM. Rapid self-tests will be made available at check-in for those who would like to test themselves for COVID-19 during the meeting.
+
+*Testing Positive for COVID-19*
+
+Individuals who are COVID-positive should not attend the UGM. If you receive a positive COVID-19 test result during the UGM, please notify members of the MDAnalysis UGM organizing committee and do not return to the event unless you have been cleared by the committee.
```

Comment:
```suggestion
Individuals who are COVID-positive should not attend the UGM. If you receive a positive COVID-19 test result during the UGM, please [notify members of the MDAnalysis UGM organizing committee][some email address here?] and do not return to the event unless you have been cleared by the committee.
```
We need to signpoint what the email address should be.

Reply:
Do we have a criteria for clearing folks? Otherwise it's going to be kinda difficult.

Reply:
We already have a ugm@mdanalysis.org google group set up for people to ask questions. Is everyone okay with me changing the permissions on this email so that anyone can post, but only group owners can read the messages?

Reply:
Yes, I think ugm@mda is a good address for such things. It's only a small group who has access, right? Because COVID is health data and one should be careful with it.

Reply:
I tested with my personal gmail account, and it looks like ugm@mdanalysis.org can't be found by the public. The only people with access are @IAlibay @orbeckst @micaela-matta @richardjgowers and @jennaswa.